clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -67,7 +67,9 @@ class ElasticsearchIndex[+CI <: ClioIndex](
           case s if s.isString =>
             s.asString
           case a if a.isArray || a.isObject =>
-            throw new RuntimeException("Arrays and objects cannot be used as ID fields")
+            throw new RuntimeException(
+              s"Found '$a' for key $key in index $indexName. Arrays and objects cannot be used as ID fields"
```

Comment:
Can you add the upsert ID to the message, too? That way we can track down the document in storage to see what's up.